helm/idol-library-example-test/values.yaml

```
@@ -125,6 +125,17 @@ ingress:
   # e.g. https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations
   annotations: {}
 
+  # -- Whether ingress uses TLS. You must set an ingress host to use this. 
+  tls:
+    # -- The name of the secret for ingress TLS. Leave empty if not using TLS. 
+    secretName: ""
+
+    # -- Certificate to use for tls
+    crt: ""
+
+    # -- Private key to use for tls
+    key: ""
+
```

```suggestion
    # -- Whether ingress uses TLS. You must set an ingress host to use this. 
    # See https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
  tls:
    # -- The name of the secret for ingress TLS. Leave empty if not using TLS. 
    # If specified then either this secret must already exist, or crt and key values must be provided.
    secretName: ""

    # -- Certificate data value to use for tls (should be base64 encoded)
    crt: ""

    # -- Private key data value to use for tls (should be base64 encoded)
    key: ""

```
Suggestion to improve docs